client/seal_direct_test.go

```diff
@@ -0,0 +1,264 @@
+package client
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"testing"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+
+	"github.com/google/go-tpm-tools/internal/test"
+	"github.com/google/go-tpm/direct/structures/tpm"
+	"github.com/google/go-tpm/direct/structures/tpml"
+	"github.com/google/go-tpm/direct/structures/tpms"
+	"github.com/google/go-tpm/direct/structures/tpmt"
+	tpm2direct "github.com/google/go-tpm/direct/tpm2"
+)
+
+func CreatePCRSelection(s []int) ([]byte, error) {
```

Comment:
we can have this be an unexported util (e.g., `createPCRSelect`) for direct returning the []bytes that represent a PCRSelect. The util above (`createTPMLPCRSelection`) can then use this func. the two utils should go in the same util file.

Reply:
See above.